### DISCUSSION TITLE
Not extracted highlights

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#8704] [Q&A]
For some reason, after last updates (v2021.12.1-29 and newer), highlights are not extracted from one of my books (in pdf format). The process takes place, but I cannot find the file in the clipboard folder. This problem doesn't exist for other pdf files.

Any idea for the reason?

**Comment by Frenzie** [comment]
Pinging @hius07 who I think is probably most familiar with the details of that right now.

**Reply by ichnilatis-gr** [reply]
I've mentioned this problem also in #8601 issue ([#8601 (comment)](https://github.com/koreader/koreader/issues/8601#issuecomment-1008864348)). He knows.

**Comment by ichnilatis-gr** [comment]
This particular book is not detected by "KOHighlights" either.
Any idea what might be going on, @hius07?

**Reply by hius07** [reply]
No ideas, I can try to look into its metadata file.

**Reply by ichnilatis-gr** [reply]
I had given it to you. Do you still have it or should I upload it?

**Reply by hius07** [reply]
Try the following: open the metadata file in any text editor and remove keys `bookmarks_sorted` and `bookmarks_sorted_20220106`
(make a backup copy of the file before that).
Then open the book and open the bookmark list. Then try to export and KOHighlights.

**Reply by ichnilatis-gr** [reply]
Thank you for the reply. I'll try it.
I don't know if it's worth mentioning that I have deleted the highlights that were saved in the pdf, as I have in the meantime disabled the relevant option.

**Reply by ichnilatis-gr** [reply]
I deleted these two lines, but the book didn't open. I changed the value to `false` but nothing changed; the highlights are not extracted.

**Reply by hius07** [reply]
Are there any other problems with bookmarks? Can you add new highlights, notes, remove, search?
Do you see the correct number of bookmarks in the Bookmark list (tap on the left title icon)?

**Reply by ichnilatis-gr** [reply]
Everything is OK with bookmarks. I can add, remove etc. I just can't extract them. The notification about the process of extraction appears, that tells that I can find the file in clipboard folder, but nothing can be found in it... A real puzzle...

**Reply by hius07** [reply]
So need to wait for https://github.com/koreader/koreader/issues/8755, and then fix further issues in the exporter plugin.
